main.cc

```diff
@@ -219,46 +198,40 @@ print_extended_info(void)
 int
 main(int argc, char* argv[])
 {
+#ifdef AFL_INPUT_FUZZING
+  AFL_INIT_ARGV();
+#endif
   int c;
   int argn;
   ff_vecs_t* ivecs = NULL;
   ff_vecs_t* ovecs = NULL;
   filter_vecs_t* fvecs = NULL;
-  char* fname = NULL;
-  char* ofname = NULL;
+  QString fname = NULL;
+  QString ofname = NULL;
```

Comment:
These look like awkward constructor semantics for QString. According to the documentation, a default QString object is null, so just "QString ofname;" should suffice.